things/database.py

```diff
@@ -26,6 +26,8 @@
 
 # Translate app language to database language
 
+BOOL_TO_FILTER = {True: "1", False: "0"}
```

Comment:
For consistency, I'd propose doing this instead:

```python
TRASHED_TO_FILTER = {False: "trashed = 0", True: "trashed = 1"}
```

This lets you pass in `trashed = None` as well, which will include both trashed and non-trashed elements.